src/reducers/tagInvitationEditor.js

```diff
@@ -1,14 +1,18 @@
 import { UPDATE_TAG_INVITATION_EDITOR, CLOSE_MODAL, SEND_COMMUNITY_TAG_INVITATION } from '../actions'
 import { partition, some } from 'lodash'
 
-export default function (state = {}, action) {
+const defaultState = {
```

Comment:
curious about your thoughts on this. I don't think we do this anywhere, but it simplifies some of the code in ShareTopicModal. Any objections?